sub.js

```diff
@@ -0,0 +1,6 @@
+const sub = (a,b) =>{
```

Comment:
@sislam16 literally blown away by your ability to declare a function. Never stop.